docs/testing-overview.md

```diff
@@ -1,29 +1,40 @@
 # Testing Overview
 
-## Snapshot testing
+Gutenberg contains both PHP and JavaScript code, and encourages testing and code style linting for both.
 
-This is an overview of [snapshot testing] and how to best leverage snapshot tests.
+## JavaScript Testing
+
+Tests for JavaScript use [Jest](http://facebook.github.io/jest/) as the test runner. If needed, you can also use [Enzyme](https://github.com/airbnb/enzyme) for React component testing.
+
+Assuming you've followed the instructions above to install Node and project dependencies, tests can be run from the command-line with NPM:
+
+```
+npm test
+```
+
+To run unit tests only, use `npm run test-unit` instead.
 
```

Comment:
Should we make it more clear that this bypasses the linter?

To run unit tests only, without the linter, use `npm run test-unit` instead.

Reply:
👍